src/latex/LaTeXWriter.jl

```diff
@@ -391,40 +411,48 @@ function latex(io::Context, node::Node, contents::Documenter.ContentsNode)
         _println(io, "}")
     end
     # print any remaining missing \end{itemize} statements
-    for _ = 1:depth; _println(io, "\\end{itemize}"); end
+    for _ in 1:depth
+        _println(io, "\\end{itemize}")
+    end
     _println(io)
+    return
 end
 
 function latex(io::Context, node::Node, evalnode::Documenter.EvalNode)
     if evalnode.result !== nothing
         latex(io, evalnode.result.children, toplevel = true)
     end
+    return
 end
 
 # Select the "best" representation for LaTeX output.
 using Base64: base64decode
 latex(io::Context, node::Node, ::Documenter.MultiOutput) = latex(io, node.children)
 function latex(io::Context, node::Node, moe::Documenter.MultiOutputElement)
-    Base.invokelatest(latex, io, node, moe.element)
+    return Base.invokelatest(latex, io, node, moe.element)
 end
-function latex(io::Context, ::Node, d::Dict{MIME,Any})
+function latex(io::Context, ::Node, d::Dict{MIME, Any})
     filename = String(rand('a':'z', 7))
     if haskey(d, MIME"image/png"())
         write("$(filename).png", base64decode(d[MIME"image/png"()]))
-        _println(io, """
-        \\begin{figure}[H]
-        \\centering
-        \\includegraphics[max width=\\linewidth]{$(filename)}
-        \\end{figure}
-        """)
+        _println(
+            io, """
```

Comment:
I guess I would actually expect this here:

```suggestion
            io,
            """
```